sktime/transformers/series_as_features/signature_based/transformers.py

```diff
@@ -0,0 +1,154 @@
+import numpy as np
```

Comment:
It'd make this file private (adding leading underscore) and expose the final transformer via the module's `__init__.py`